Add warning about 4.8.0 upgrade

vcerenu opened the pull request:
<!--
This template reflects sections that must be included in new Pull requests.
Contributions from the community are really appreciated. If this is the case, please add the "contribution" to properly track the Pull Request.
Please fill the table below. Feel free to extend it at your convenience.
-->
<!--
## Community contributions advice
We love our community contributions. We recommend making PRs from the current branch. For instance, if Wazuh 4.3.7 is the latest release, the branch to be used is 4.3.
Thanks!
-->
## Description
<!--
Add a clear description of how the problem has been solved.
If your PR closes an issue, please use the "closes" keyword indicating the issue.
-->
This PR adds a warning into the Wazuh Kubernetes upgrade.

Related Issue https://github.com/wazuh/wazuh-documentation/issues/7122 https://github.com/wazuh/wazuh-kubernetes/pull/589

### First change

![image](https://github.com/wazuh/wazuh-documentation/assets/56500015/af0fc551-c60e-47e3-bb13-cd5aa0a1c009)

### Last change

![image](https://github.com/wazuh/wazuh-documentation/assets/56500015/dbbe60f8-5d19-4979-b603-3831d1378697)


## Checks
### Docs building
- [x] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

javimed reviewed (commented):
What if we add a step to recreate SSL ceritificates rather than warning the user about it?

I can see we currently have the following steps:

1. Checking which files are exported to the volume
2. Configuring the upgrade
3. Apply the new configuration

Maybe adding a new step with specific instructions on recreating the certificates would help. Or alternatively a step with the reference to the section on how to do this.

I don't think we should worry the user with a warning, just instruct them on how to proceed as part of the upgrade workflow.

vcerenu commented:
The change was made with a warning because the only way to explain it is to refer to the code and internal configurations of Wazuh, so instead of carrying out the step explaining it seemed more correct to add a warning so that they take it into account.

I have already modified the warning and added it as an additional step to the upgrade process, clarifying that any update from a version prior of `4.8.0` to a version equal to or later than `4.8.0 `requires it.
